notebooks/examples/analog_sensitivity_LeNet5.ipynb

```diff
@@ -0,0 +1,1483 @@
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "epochs = epochss.value\n",
+    "batchsize = batchsizes.value\n",
+    "def create_digital_network():\n",
+    "    channel = [16, 32, 512, 128]\n",
+    "    model = nn.Sequential(\n",
+    "        nn.Conv2d(in_channels=1, out_channels=channel[0], kernel_size=5, stride=1),\n",
+    "        nn.ReLU(),\n",
+    "        nn.MaxPool2d(kernel_size=2),\n",
+    "        nn.Conv2d(in_channels=channel[0], out_channels=channel[1], kernel_size=5, stride=1),\n",
+    "        nn.ReLU(),\n",
+    "        nn.MaxPool2d(kernel_size=2),\n",
+    "        nn.ReLU(),\n",
+    "        nn.Flatten(),\n",
+    "        nn.Linear(in_features=channel[2], out_features=channel[3]),\n",
+    "        nn.ReLU(),\n",
+    "        nn.Linear(in_features=channel[3], out_features=N_CLASSES),\n",
+    "        nn.LogSoftmax(dim=1)\n",
+    "    )\n",
+    "\n",
+    "    return model\n",
+    "\n",
+    "\n",
+    "train_data, validation_data = load_images(batchsize)\n",
+    "\n",
+    "# Prepare the model.\n",
+    "dmodel = create_digital_network()\n",
+    "dmodel.to(DEVICE)\n",
+    "\n",
+    "optimizer = create_digital_optimizer(dmodel, LEARNING_RATE)\n",
+    "\n",
+    "criterion = nn.CrossEntropyLoss()\n",
+    "\n",
+    "fig = plt.figure(figsize=(5, 4), dpi=150)\n",
+    "dmodel, optimizer, _ = training_loop(dmodel, criterion, optimizer, train_data, validation_data, epochs, fig)\n",
+    "plt.show()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "30588194",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Accuracy of the digital model: 96.02%\n"
+     ]
+    }
+   ],
+   "source": [
+    "dmodel, _, _, digital_accuracy = test_evaluation(validation_data, dmodel, criterion)\n",
+    "print(f\"Accuracy of the digital model: {digital_accuracy:.2f}%\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "14ceee0f",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Experiment 1: Analog Inference\n",
+    "\n",
+    "As a preliminary experiment, we perform \"analog\" inference of the previously trained digital LeNet with a floating point synaptic device model, while ignoring the non-idealities of peripheral circuits. "
```

Comment:
This is indeed just the floating point model if you use the FloatingPointRPUConfig. Could you maybe add a sentence say this? Something like: Using FloatingPointRPUConfig is a way to run the floating point reference model using the "analog" layers and should result in the same accuracy as the model with native pytorch layers.